Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -595,9 +595,10 @@ def is_usable(self, force_update=False):
         if not service:
             _log.debug(u'{} has no service to check if the device is usable'.format(self.device_type.software_variant))
             return True
-
-        exit_code = self.executive.run_command([SimulatedDeviceManager.xcrun, 'simctl', 'spawn', self.udid, 'launchctl', 'list', service], return_exit_code=True)
+        for service in service:
```

Comment:
Is there a reason you're iterating through multiple services here? We only have on defined per device type.

Reply:
I sent this to Jonathan - I could technically drop the for loop (Though leaving it allows for additional apps in the future and whatnot.)... So we could get rid of it; but leaving it allows for additional options if someone wants to modify the method in the future. Either is fine with me.

Reply:
Seeing this in total, I think getting rid of the loop is probably best.

If you wanted to retain the loop, the terminate would need to be in the loop, then the early return after the loop. But it's even a bit weirder than that, because in the loop case, you all listed applications to start, so it's likely that you're going to end up starting some services multiple times. If you were doing the loop, you should also early return on error. ie, if the first process failed to start, then we shouldn't even try the second.